tests/gbs/test_sample.py

```diff
@@ -58,7 +60,7 @@
     [0.15938187, 0.10387399, 1.10301587, -0.26756921, 0.32194572, -0.24317402, 0.0436992]
 )
 
-p = [U1, r, U2, alpha]
+p = [t, U1, r, U2, alpha]
```

Comment:
We should also test for the nonzero squeezing case.

How about we have a p0 and a p1 which have a zero and nonzero two mode squeezing.

we can then do something like:
`@pytest.mark.parametrize("p", [p0, p1])`
above the tests.

Note the `test_all_loss` test might need an if statement added

Reply:
The difference in T=0 and T!=0 samples is the absence of photons in the last N modes of the zero temperature case. Since majority of the modes might already have zero photons in them, any test will be probabilistic and probably not very useful. Am I missing something here?